bindings/julia/Displaz.jl

```diff
@@ -107,25 +155,27 @@ function plot(position; color=[1 1 1], markersize=[0.1], markershape=[0])
     size(color,1) == nvertices || error("color must have same number of rows as position array")
     #fileName = "_julia_tmp.ply"
     fileName = tempname()*".ply"
-    write_ply_points(fileName, nvertices, (
-                     (:position, vector_semantic, position),
-                     (:color, color_semantic, color),
-                     (:markersize, array_semantic, markersize),
-                     (:markershape, array_semantic, markershape),
-                     ))
-    hold = _hold ? "-add" : []
+	if isequal(markershape[1],5) # Plot lines
```

Comment:
This could probably just be

`if '-' in markershape`

Then it'd also work when markershape is a `String` of some sort. and be less obscure as well.